src/web/app/pages-instructor/instructor-session-modal-page.component.ts

```diff
@@ -161,10 +164,11 @@ export abstract class InstructorSessionModalPageComponent extends InstructorSess
               isSelected: selectAllRespondents && !giverSet.has(instructor.email),
             } as InstructorListInfoTableRowModel));
 
-        modalRef.result.then((respondentsToRemind: any[]) => {
+        modalRef.result.then((reminderResponse: ReminderResponseModel) => {
           this.isSendReminderLoading = true;
           this.feedbackSessionsService.remindFeedbackSessionSubmissionForRespondents(courseId, feedbackSessionName, {
-            usersToRemind: respondentsToRemind.map((m: any) => m.email),
+            usersToRemind: reminderResponse.respondentsToSend.map((m: any) => m.email),
+            sendCopyToInstructor: reminderResponse.isSendingCopyToInstructor,
```

Comment:
any particular reason the naming is different? There might be benefits (although probably marginal in this case) of using a consistent name for both the frontend and backend.

Reply:
Sounds like a case of intent-perception gap :) 

Didn't mean to imply that you have to use that name, please choose any suitable name for this!